### PR TITLE
#4790 [Bug] When the topic is running, the workspace restart will fail

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4790 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 1]
```diff
@@ -208,6 +210,18 @@ const stopWorker$ = params =>
...
+const stopTopic$ = topic =>
+  of(
+    defer(() => topicApi.stop(topic)).pipe(
+      delay(duration),
+      map(() => actions.stopTopic.request()),
+      catchError(error => of(actions.stopTopic.failure(error))),
+    ),
+  ).pipe(concatAll());
+
+const stopAllTopics$ = topics =>
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:222]:
```suggestion
const stopTopics$ = topics =>
```

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 2]
```diff
@@ -244,6 +258,18 @@ const startWorker$ = (params, skipList) =>
...
+const startTopic$ = topic =>
+  of(
+    defer(() => topicApi.start(topic)).pipe(
+      delay(duration),
+      map(() => actions.startTopic.request()),
+      catchError(error => of(actions.startTopic.failure(error))),
+    ),
+  ).pipe(concatAll());
+
+const startAllTopics$ = topics =>
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:270]:
```suggestion
const startTopics$ = topics =>
```

File: ohara-manager/client/src/components/Workspace/Settings/Settings.js [hunk 3]
```diff
@@ -45,7 +46,28 @@ const Settings = () => {
   const tmpWorker = hooks.useWorker();
   const tmpBroker = hooks.useBroker();
   const tmpZookeeper = hooks.useZookeeper();
+  const connectors = hooks.useConnectors();
+  const topics = hooks.useAllTopics();
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply ohara-manager/client/src/components/Workspace/Settings/Settings.js]: ok

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**